### PR TITLE
More build improvements

### PR DESCRIPTION
**Pull request by robmen**

### CODE REVIEW

File: src/internal/SetBuildNumber/Directory.Packages.props.pp [hunk 1]
```diff
@@ -42,6 +42,7 @@
     <PackageVersion Include="System.Text.Encoding.CodePages" Version="4.6.0" />
 
     <PackageVersion Include="Microsoft.AspNetCore.Owin" Version="3.1.13" />
+    <PackageVersion Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" />
```
Comment (rseanhall) [inline src/internal/SetBuildNumber/Directory.Packages.props.pp:45]: Just remove this and the references to this package everywhere. They're only causing problems with no benefit.

Comment (robmen) [reply src/internal/SetBuildNumber/Directory.Packages.props.pp]: I really wanted to understand what was going on here... but I give up.

File: src/Directory.Build.rsp [hunk 2]
```diff
@@ -0,0 +1,4 @@
+-nologo
+-nr:false
+-v:m
```
Comment (rseanhall) [inline src/Directory.Build.rsp:3]: 1. When are these used?
2. Isn't -nologo redundant? It seems to be specified everywhere already.
3. How do you get the binlogs from the CI server?

File: src/internal/SetBuildNumber/Directory.Packages.props.pp (hunk 1 above)
Comment (rseanhall) [reply src/internal/SetBuildNumber/Directory.Packages.props.pp]: I assume we're hitting another edge case in Nuget that no one else cares about.

File: src/Directory.Build.rsp (hunk 2 above)
Comment (robmen) [reply src/Directory.Build.rsp:3]: 1. Every build.
2. There is a bug open in MSBuild to respect `-nologo` in the response files. So for now it has to be added everywhere.
3. The "Summary" of the build shows all the uploaded artifacts. `Logs_#####` is one of them. You can "Open" that then double click the .binlogs inside if you have binlog viewer installed. Better experience than I expected.

File: src/internal/SetBuildNumber/Directory.Packages.props.pp (hunk 1 above)
Comment (robmen) [reply src/internal/SetBuildNumber/Directory.Packages.props.pp]: Yeah, CPVM is still "preview" so part of it is my own doing but I was hoping I could understand the root issue to possibly better handle something like it in the `wix.targets` in the future.

But you're probably right that it is a collection of edge cases no one really cares about.

File: src/Directory.Build.rsp (hunk 2 above)
Comment (rseanhall) [reply src/Directory.Build.rsp:3]: I feel like we need binlogs for everything if everything is building with minimal verbosity.

Comment (robmen) [reply src/Directory.Build.rsp:3]: We need binlogs for all builds.

**Pull request merged**